attribute_set_mass_edit/wizards/mass_editing_wizard.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, models
+from odoo.fields import first
+
+
+class MassEditingWizard(models.TransientModel):
+    _inherit = "mass.editing.wizard"
+
+    @api.model
+    def _get_field_options(self, field):
+        res = super(MassEditingWizard, self)._get_field_options(field)
+        if field.ttype in ("many2one", "many2many"):
+            attributes = self.env["attribute.attribute"].search(
+                [("field_id", "=", field.id)]
+            )
+            attribute = first(
+                attributes.filtered(
+                    lambda a: a.attribute_type in ("select", "multiselect")
+                )
+            )
```

Comment:
@rousseldenis can't we do simply this?
```suggestion
            attribute = self.env["attribute.attribute"].search([
                ("field_id", "=", field.id),
                ("attribute_type", "in", ("select", "multiselect"))
            ], limit=1)
```